manuscript/chapter2.md

```diff
@@ -1339,17 +1340,17 @@ class Search extends Component {
 }
 ~~~~~~~~
 
-The "Search" text should now be visible next to your input field. And, when you use the Search component somewhere else, you can, if you like, use different text. After all, it's not only text that can be passed as children. You can pass an element, or element trees (which can be encapsulated by components again), as children. The children property makes it possible to weave components into each other.
+The "Search" text should now be visible next to your input field. When you use the Search component elsewhere, you can use different entities, since it's not just text that can be passed as children. You can also pass an element, or element trees that can be encapsulated by components, as children. The children property makes it possible to weave components into each other.
 
 ### Exercises:
 
-* read more about [the composition model of React](https://reactjs.org/docs/composition-vs-inheritance.html)
+* Read more about [the composition model of React](https://reactjs.org/docs/composition-vs-inheritance.html)
 
 ## Reusable Components
 
-Reusable and composable components empower you to come up with capable component hierarchies. They are the foundation of React's view layer. The last chapters mentioned reusability. Now, you can reuse the Table and Search components. Even the App component is reusable, as you could instantiate it somewhere else again.
+Reusable and composable components empower you to come up with capable component hierarchies, the foundation of React's view layer. The last chapter mentioned reusability, and now we can see how reusing the Table and Search components to create more functionality. Even the App component is reusable, as it can be instantiated elsewhere as well.
```

Comment:
Maybe: "The last sections mentioned ..." instead of "The last chapter mentioned ..."